app/screenshots.gradle

```diff
@@ -23,7 +23,7 @@ android {
             managedDevices.devices {
                 Docs(ManagedVirtualDevice) {
                     device = "Pixel"
-                    apiLevel = 31
+                    apiLevel = 30
```

Comment:
INSTRUMENTATION_ABORTED: System has crashed.

Had to migrate to API 30. At least it doesn't produce transparent screenshots anymore

Reply:
rather than downgrading we should try to upgrade our screenshot emulator to be on the latest version. if we're stuck on an older version because of ktor we might not want to migrate to ktor

Reply:
Yep, 31 is the latest available version. And we have to use 30 otherwise the tests might crash

Reply:
then maybe ktor isn't the right way to go if it forces us to downgrade. the further we fall behind the worse

Reply:
Reverted. The issue was related to my local gradle-managed device setup.